lib/network/base/base_api_ds.dart

```diff
@@ -0,0 +1,70 @@
+import 'dart:convert';
```

Comment:
Can we move network classes into different framework/dependency and then we can use via pubspec.